apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -177,6 +178,56 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
             />
           }
         />
+        <Route
+          path={TEACHER_NAVIGATION_PATHS.lessonMaterials}
+          element={
+            <ElementOrEmptyPage
+              showNoStudents={studentCount === 0}
+              showNoCurriculumAssigned={!anyStudentHasProgress}
+              element={applyV1TeacherDashboardWidth(<TemporaryBlankPage />)}
+            />
+          }
+        />
+        <Route
+          path={TEACHER_NAVIGATION_PATHS.calendar}
+          element={
+            <ElementOrEmptyPage
+              showNoStudents={studentCount === 0}
+              showNoCurriculumAssigned={!anyStudentHasProgress}
+              element={applyV1TeacherDashboardWidth(<TemporaryBlankPage />)}
+            />
+          }
+        />
+        <Route
+          path={TEACHER_NAVIGATION_PATHS.courseOverview}
+          element={
+            <ElementOrEmptyPage
+              showNoStudents={studentCount === 0}
+              showNoCurriculumAssigned={!anyStudentHasProgress}
+              element={applyV1TeacherDashboardWidth(<TemporaryBlankPage />)}
+            />
+          }
+        />
+        <Route
+          path={TEACHER_NAVIGATION_PATHS.unitOverview}
+          element={
+            <ElementOrEmptyPage
+              showNoStudents={studentCount === 0}
+              showNoCurriculumAssigned={!anyStudentHasProgress}
+              element={applyV1TeacherDashboardWidth(<TemporaryBlankPage />)}
+            />
+          }
+        />
+        <Route
+          path={TEACHER_NAVIGATION_PATHS.settings}
+          element={
+            <ElementOrEmptyPage
```

Comment:
Settings should not show the `ElementOrEmptyPage`, I think the logic for course and unit overview might need to change too.

Reply:
Can you say more about this?  Are you suggesting that because the `SectionsSetupContainer` is structured differently than the rest of the "elements" that we will want to render `SectionsSetupContainer` directly here?

Either way, I'd like to advocate for leaving this as is for now, to get this PR in and be able to get feedback from Bryan/Molly as we clean up the rest of the pages and content.  Do you have any concerns about that plan?

Reply:
Fine to leave as is for now, but we will need to remove it anyways.

The `ElementOrEmpty` page will show an error when a user attempts to navigate to a page. We don't want the user to see that error when they are trying to access some pages, including manage students and settings. This is because we want the user to be able to add students and edit their section details even if they don't have curriculum assigned or students in the section yet